common/hooks/useTranslationNamespace.ts

```diff
@@ -0,0 +1,10 @@
+import { useTranslation } from 'next-i18next';
+import usePath from './usePath';
+import { capitalize } from '../utils/capitalize';
+
+export function useTranslationNamespace(block: string) {
+  const { slicePathname } = usePath();
+  const { t } = useTranslation(`${block}${capitalize(slicePathname)}`);
```

Comment:
I think all methods of this hook "useTranslation" should be returned here.

It would be better to assign a constant to this hook and return it from the component